theme.js

```diff
@@ -0,0 +1,43 @@
+import { slug } from 'github-slugger'
+import path from 'node:path'
+import {
+  MarkdownTheme,
+  MarkdownThemeRenderContext
+} from 'typedoc-plugin-markdown'
+
+const externalLinkRegex = /^(http|ftp)s?:\/\//
+
+export function load(app) {
+  app.renderer.defineTheme('custom-markdown-theme', CustomMarkdownTheme)
+}
+
+export class CustomMarkdownTheme extends MarkdownTheme {
+  getRenderContext(pageEvent) {
+    return new CustomMarkdownThemeContext(pageEvent, this.application.options)
+  }
+}
+
+class CustomMarkdownThemeContext extends MarkdownThemeRenderContext {
+  relativeURL = url => {
+    if (!url) {
+      return null
+    } else if (externalLinkRegex.test(url)) {
+      return url
+    }
+
+    let basePath = this.options.getValue('basePath')
+    let basePathParsed = path.parse(basePath)
```

Comment:
My main concern was that `path.parse()` on Windows will not work with UNIX-style `/`.

But maybe we need a proper Windows test later.

If you think that it will works, let’s keep it.

Reply:
I've tested it on real Windows machine